my_list.rb

```diff
@@ -0,0 +1,13 @@
+require_relative 'my_enumerable'
+
+class MyList
+  include MyEnumerable
+
+  def initialize(*args)
+    @list = args
+  end
+
+  def each(&block)
+    @list.each(&block)
+  end
```

Comment:
- [ ] Kindly include the **yield** method. MyList class must implement #each method that yields all members of @list in succession. Here is more information about the [yield](https://www.theodinproject.com/lessons/ruby-blocks) method in ruby.

Reply:
Ok but if i use yield method it shows me this linter error:
```
my_list.rb:11:5: C: Style/ExplicitBlockArgument: Consider using explicit block argument in the surrounding method's signature over yield.
    @list.each { |item| yield(item) }
```